src/m_atm.cc

```diff
@@ -340,3 +342,57 @@ void atm_fieldAddNumericData(AtmField &atm_field, const QuantumIdentifier &key,
                      "Cannot add numeric data to regularized atmospheric field")
   atm_field[key] = data;
 }
+
+void atm_fieldIGRF(AtmField &atm_field, const Time &time, const Verbosity &) {
+  using namespace IGRF;
+  static const Vector ell{6378137., 0.081819190842621};
```

Comment:
Instead of using "magic" numbers, it would be nice to use Earth radius from our constants. And what is the second number?

Reply:
@riclarsson If the second number is flattening, please note that I have changed the definition of refellipsoid to instead have polar radius as second element. The new version is so far called refellipsoidZZZ:

*-------------------------------------------------------------------*
Workspace variable = refellipsoidZZZ
---------------------------------------------------------------------

Reference ellipsoid.

This vector specifies the shape of the reference ellipsoid. The
vector must have length 2, where the two elements are:
  1: Equatorial radius.
  2: Polar radius.

Usage:  Set by the user.

Size:   [ 2 ]


---------------------------------------------------------------------
Group = Vector
*-------------------------------------------------------------------*

Reply:
@erikssonpatrick Then I will update the code when we make the switch.  I think it is relatively easy in the underlying code, but it is used from parts that takes `refellipsoid` so I cannot do that update before  :)

@olemke It is not a magic number per se.  IGRF works in spherical coordinates.  I need the exact radius to a point.  We only allow two components in ellipsoid, so this is the closest I can get to give a good "radius" as input.  Since IGRF is a model, it works on its own idea of Earth and not on the user's idea of Earth

Reply:
In that case a comment explaining what those two number are would be appropriate.